### PR TITLE
swagger: update dvir list

### PR DESCRIPTION
**Pull request by dsharrison**
Add the new id and timeMs properties to the DVIR list response (https://github.com/samsara-dev/backend/pull/14219).

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -3369,6 +3369,17 @@
               "items": {
                 "type": "object",
                 "properties": {
+                  "id": {
+                    "type": "integer",
+                      "format": "int64",
```
Comment (silverspace) [inline swagger.json]: Nit: formatting looks slightly off.

File: swagger.json [hunk 2]
```diff
@@ -3369,6 +3369,17 @@
...
+                      "description": "The id of this DVIR record.",
+                      "example": 19
+                  },
+                  "timeMs": {
+                    "type": "integer",
```
Comment (silverspace) [inline swagger.json]: Nit: could we also include `"format": "int64"` here?

File: swagger.json (hunk 1 above)
Comment (dsharrison) [reply swagger.json]: 🤦‍♂️ I think my vscode settings for this project are messed up, thanks for calling that out!

File: swagger.json (hunk 2 above)
Comment (dsharrison) [reply swagger.json]: Sure!

File: swagger.json (hunk 1 above)
Comment (dsharrison) [reply swagger.json]: fixed!

File: swagger.json (hunk 2 above)
Comment (dsharrison) [reply swagger.json]: fixed!

**Comment by dsharrison** [comment]
Fixed those nits. Maybe we want to add a checkbox or step to the PR description template for verifying swagger rendering to remind people to confirm they have done that step?

**Pull request merged**